src/main/java/salesforece/ui/pages/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package salesforece.ui.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * This class return a instance of LoginPage.
+ */
+public class LoginPage extends BasePage {
+
+    @FindBy(id = "username")
+    private WebElement userNameTxtBox;
+
+    @FindBy(id = "password")
+    private WebElement passwordTxtBox;
+
+    @FindBy(id = "Login")
+    private WebElement loginBtn;
+
+    public LoginPage(WebDriver driver) {
```

Comment:
I think Javadoc is missing for the constructor

Reply:
Checkstyle rules say javadocs in the constructor is not necessary, but i'm not sure